OneSignalSDK/onesignal/src/main/java/com/onesignal/OSPreferencesWrapper.java

```diff
@@ -0,0 +1,81 @@
+package com.onesignal;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.Set;
+
+class OSPreferencesWrapper implements OSSharedPreferences {
+
+    @Override
+    public String getOutcomesV2KeyName() {
+        return OneSignalPrefs.PREFS_OS_OUTCOMES_V2;
+    }
+
+    @Override
+    public String getPreferencesName() {
+        return OneSignalPrefs.PREFS_ONESIGNAL;
+    }
```

Comment:
Lets add a global version number and increment it so we only need one
`getPreferencesName`

Reply:
Sorry didn't understand this one

Reply:
Like for SQL we have a DB version, what if we have a Shared Prefs version? Idk just a thought

Reply:
oh gotcha, this isn't a version though is only the preference name. I was thinking of having all by default saved on OneSignalPrefs.PREFS_ONESIGNAL preference, but this will need more refactoring. I can create a ticket if you agree

Reply:
Its not a big deal, was just saying we should use the DB version to know which name to use to get the shared prefs